src/Timer.ts

```diff
@@ -156,10 +163,17 @@ export default class Timer implements Readable<TimerStore> {
     private timeup() {
         let autostart = false
         this.update((state) => {
-            const ctx = this.createLogContext(state)
-            this.processLog(ctx)
-            autostart = state.autostart
-            return this.endSession(state)
+            this.notify(state)
+            if (!s.continueAfterZero) {
```

Comment:
```suggestion
            if (!state.continueAfterZero) {
```

Got error in console that S didn't exist, swapping it for state did the trick :)